build.gradle

```diff
@@ -5,7 +5,7 @@ subprojects {
     apply plugin: 'signing'
 
     group = "com.google.cloud.opentelemetry"
-    version = "0.0.0-SNAPSHOT" // CURRENT_VERSION
+    version = "0.8.0-SNAPSHOT" // CURRENT_VERSION
```

Comment:
Shouldn't it be `v0.6.0` based on the current dependency version https://github.com/GoogleCloudPlatform/opentelemetry-operations-java/blob/ffcdeb99466b848ae1138b68ce431187e0ac7039/build.gradle#L22 ?

Reply:
I've updated the `openTelemetryVersion` to 0.7.0 on master, so 0.8.0 should be correct now